tests/src/test/scala/com/nvidia/spark/rapids/AdaptiveQueryExecutionSuite.scala

```diff
@@ -0,0 +1,177 @@
+/*
+ * Copyright (c) 2020, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.nvidia.spark.rapids
+
+import org.scalatest.{BeforeAndAfterAll, DoNotDiscover, Suites}
+
+/**
+ * Runs all of the test suites with Adaptive Query Execution enabled.
+ */
+class AdaptiveQueryExecutionSuite extends Suites(
+  new AnsiCastOpSuiteAdaptive,
+  new CastOpSuiteAdaptive,
+  new CsvScanSuiteAdaptive,
+  new ExpandExecSuiteAdaptive,
+  new GpuBatchUtilsSuiteAdaptive,
+  new GpuCoalesceBatchesSuiteAdaptive,
+  new HashAggregatesSuiteAdaptive,
+  new HashSortOptimizeSuiteAdaptive,
+  new LimitExecSuiteAdaptive,
+  new OrcScanSuiteAdaptive,
+  new ParquetScanSuiteAdaptive,
+  new ParquetWriterSuiteAdaptive,
+  new ProjectExprSuiteAdaptive,
+  new SortExecSuiteAdaptive,
+  new StringFallbackSuiteAdaptive,
+  new WindowFunctionSuiteAdaptive)
+
+// we need the AQE suites to have unique names so that they don't overwrite
+// surefire results from the original suites
+@DoNotDiscover class AnsiCastOpSuiteAdaptive extends AnsiCastOpSuite with BeforeAndAfterAll {
```

Comment:
Style nit: Typically class annotations appear in lines above the class.  Applies to other instances in this file.